anago

```diff
@@ -970,7 +970,11 @@ announce () {
     logecho "$WARNING: Email cannot be sent from GCB.  Execute the following" \
             "after a completed release to send email notification:"
     logecho
-    logecho "$ release-notify $RELEASE_VERSION_PRIME"
+    if ((FLAGS_nomock)); then
```

Comment:
Style-wise, I prefer to declare this up above:

```
local nomock_flag
((FLAGS_nomock)) && nomock_flag="--nomock"
```
And then use it in-line (And while we're at it, use $PROG too)...
```
logecho "$ $PROG $nomock_flag $RELEASE_VERSION_PRIME"
```

Reply:
Fixed!